data/basejob_schema.json

```diff
@@ -0,0 +1,11 @@
+{
+    "type": "object",
+    "properties": {
+      "jobname": {
+        "type": "string"
+      },
+      "partition": {
+        "type": "string"
+      }
+    }
+}
```

Comment:
Are these properties required? You can add in a required key

```javascript
required: ["jobname", "partition"]
```

And are additional properties that aren't listed allowed? For example, could I add in something like "hello" as a key into the object?

If not, then you can also add in 
```
additionalProperties: {
    not: true
}
```

Reply:
Thanks for the hints!. I've added more details, `defaults`, `min`, `max`, types etc.
I'm not sure yet how to leverage them for the validation (and it's connection to Vuex), but I'll look more into it